admin/meetings/111624-sprint-1-retrospective.md

```diff
@@ -0,0 +1,42 @@
+# 11/16/2024 - Retrospective Meeting
+
+Attendees: Tanmay, Mokshita, Kshitij, Angus, Chris, Fude, Jyotika, Rohan, Chris C
+
+
+
+## Liked
+* Everyone's participation
+* Pitch planning meeting 
+* Good Progress on the analysis tasks
+* Enthusiasm in team members
+* Everything is done on time.
+* Tasks are organized
+* Clear work seperation
+* Finish tasks on time
+* Switching to Discord
+* Great communication
+* Speed at which the meeting times are decided
+
+## Learnt
+* Github issues and actions using
+* Brownfield projects 
+* Code Quality Tools
+* Github Actions, Documentation tools
+* How to document properly
+* What's linting and there are many linting to choose
+* Documentation tools
+* Theres a world of VS code extension development
+
+## Lacked
+* Keep shorter meetings
+    * Resolution: Come up with a more specific agenda for the meetings
```

Comment:
This can help. 

Planning and review/retrospectives are not stand-ups, but you might find it helpful to have some during the week, so that you don't end up with what could have been brought up earlier in a stand-up getting pushed off until it turns into a longer discussion later on.

Keep in mind that "stand-ups" are originally just max 15m meetings where everyone stands up and says "here's what I'm working on, here's how it's going, I have these open problems". They're not really to make decisions.

Reply:
Agreed.